project-backend/Providers/BidProvider/BidProvider.cs

```diff
@@ -0,0 +1,60 @@
+﻿using project_backend.Models.Bid;
+using project_backend.Models.Exceptions;
+using project_backend.Models.Worker;
+using project_backend.Repos;
+using System;
+using System.Linq;
+namespace project_backend.Providers.BidProvider
+{
+    public class BidProvider : IBidProvider
+    {
+
+        private readonly DatabaseContext _dbContext;
+
+        public BidProvider(DatabaseContext databaseContext) =>
+            _dbContext = databaseContext;
+
+        public BidDAO AddBid(float sum, int workerId, int jobId)
+        {
+            WorkerDAO worker = _dbContext.Workers.Find(workerId);
+
+            if (worker == null)
+            {
+                throw new NotQualifiedException();
+            }
```

Comment:
This should also be changed after implementing the worker class and ROLE in the auth. Works for now.